examples/plot_wlasso.py

```diff
@@ -10,90 +10,96 @@
 
```

Comment:
In the docstring say that you compare  LassoCV (a single parameter) with wLasso with sparse ho for tuning